velox/docs/functions/presto/conversion.rst

```diff
@@ -268,6 +268,27 @@ Invalid examples
   SELECT cast('nan' as bigint); -- Invalid argument
   SELECT cast('infinity' as bigint); -- Invalid argument
 
+From decimal
+^^^^^^^^^^^^
+
+Casting from decimal input to an integral type will check cast_to_int_by_truncate configuration.
```

Comment:
See https://github.com/facebookincubator/velox/pull/5844 for comments on wording.